backend/trafficrules/src/main/java/com/coffeejug/trafficrules/model/User.java

```diff
@@ -1,18 +1,33 @@
 package com.coffeejug.trafficrules.model;
 
+import java.time.LocalDateTime;
 import java.util.Objects;
 import java.util.UUID;
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
 import javax.persistence.Table;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
```

Comment:
`@JsonIgnore` annotation is referred to UI and it's not recommended to have in the entity class